docs/app/pages/components/sections/date-time-picker/time-picker/time-picker.component.html

```diff
@@ -0,0 +1,122 @@
+<timepicker [ngModel]="myTime" [min]="minTime" [max]="maxTime" [hourStep]="hstep" [minuteStep]="mstep" [showMeridian]="ismeridian"
```

Comment:
Do you want me to change it for all lines which exceed the limit or just this one?
Probably a stupid question, but I just want to make sure.